.gitlab-ci.yml

```diff
@@ -0,0 +1,92 @@
+default:
+  image:
+    name: gcr.io/kaniko-project/executor:debug
+    entrypoint: [ "" ]
+  tags:
+    - ord1-tenant
+
+stages:
+  - test
+  - generate
+  - build
+  - release
+
+variables:
+  CONTROLLER_FIXED_IMAGE_TAG: ${CI_COMMIT_SHORT_SHA}
+
+.go-cache:
+  variables:
+    GOPATH: $CI_PROJECT_DIR/.go
+  before_script:
+    - mkdir -p .go
+  cache:
+    paths:
+      - .go/pkg/mod/
+
+test:
+  stage: test
+  image: golang:1.18-alpine3.15
+  extends: .go-cache
+  cache:
+    key: bin-${CI_COMMIT_REF_SLUG}
+    paths:
+      - bin/
+  script:
+    - go test ./... -coverprofile cover.out
+
+build:image:
+  stage: build
+  variables:
+    DOCKER_IMG: ${CONTROLLER_IMAGE}
```

Comment:
Is `CONTROLLER_IMAGE` set in CI args?

Reply:
ah that is old i need to remove one sec.